formhero/forms/tests.py

```diff
@@ -1,4 +1,32 @@
-from django.test import TestCase
+from django.contrib.auth import get_user_model
+from formhero.apps.models import App
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+from formhero.forms.views import CreateForm
+from rest_framework import status
 
 
-# Create your tests here.
+User = get_user_model()
+
+
+class RetrievePostFormTests(APITestCase):
+
+    def test_post_a_new_form(self):
```

Comment:
@szymonzmyslony This should be called `test_should_create_new_form_when_valid_data_is_posted()`
From the Newcomers Guide:

> To aid naming tests, our convention is to start each test with should_, resulting in test names matching the pattern test_should_<action>_<when/if>_<condition>

https://dangerfarms.gitbooks.io/newcomers-guide/content/section-4/django/